cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -625,7 +615,6 @@ policy "pol_utilization" do
     summary_template "AWS Account ID: {{with index data.result 0}}{{ .accountId }}{{end}} - {{ len data.result }} rows containing AWS old snapshot data"
     detail_template <<-EOS
 The following {{ len data.result }} old snapshots, for AWS Account: {{with index data.result 0}}{{ .accountId }}{{end}}, have exceeded the specified age of: {{ parameters.snapshot_age }} days old.
-
```

Comment:
This extra newline was actually meaningful in the resulting markdown from the template, please add it back.